.travis.yml

```diff
@@ -4,11 +4,11 @@ os:
   - linux
   - osx
 julia:
-  - 0.6
+  - 0.7
   - nightly
-matrix:
-  allow_failures:
-  - julia: nightly
+# matrix:
+#   allow_failures:
+#   - julia: nightly
 notifications:
```

Comment:
Let's leave this uncommented.

Reply:
Why? `nightly` is closer to `v0.7.0` than `v0.7-beta` (which the 0.7 travis target points to).

Reply:
Wait, `-nightly` and `-0.7` are the same right now for Travis?

I feel that's it bad form to predicate CI pass/fail on unfixed upstream dependencies, since you can't really expect contributors to update the package for every new upstream commit. In contrast, fixing the upstream dependency to a specific tag (`v0.7-beta`, `v0.7-rc`, etc.) is a clear specification of what contributors intend to support, such that upgrading requires explicit intervention/testing.

But if  `-0.7` is actually targeting `-nightly` then that whole philosophy falls through I guess 😛

Reply:
Well, regardless, if indeed `-0.7` targets the same commits as `-nightly`, it should still be fine to uncomment `nightly` as an allowed failure in the build matrix, since the same failures would still be captured for the `0.7` runs (assuming I understand Travis' build system correctly).

Reply:
`- 0.7` points to the latest release of the 0.7 series, which currently is `0.7-beta`, but soon will be `0.7-rc1` or w/e so it is not the same as `nightly` which is the front of julia master.
Reason for commenting out `- 0.7` here is that StaticArrays is broken since it includes the `Statistics` stdlib as a dependency (my fault :stuck_out_tongue:) but that stdlib is not in the beta release, it was merged some commits after.